assets/js/modules/analytics-4/components/dashboard/DashboardAllTrafficWidgetGA4/TotalUserCount.js

```diff
@@ -144,7 +144,7 @@ export default function TotalUserCount( props ) {
 						</div>
 					) }
 					<div className="googlesitekit-data-block__change">
-						{ change && (
+						{ !! change && (
```

Comment:
Without the cast to boolean, this results in a visual `00%` where the first `0` is from this `change` which evaluates to falsey (to not render the block arrow), but is still output as expected. The `0%` is rendered correctly by `numFmt` below.